config.js

```diff
@@ -1,5 +1,9 @@
 export default {
     openAiKey: process.env.OPENAI_API_KEY,
+    azureOpenAIApiKey: process.env.AZURE_OPENAI_API_KEY,
+    azureOpenAIApiInstanceName: process.env.AZURE_OPENAI_API_INSTANCE_NAME,
+    azureOpenAIApiDeploymentName: process.env.AZURE_OPENAI_API_DEPLOYMENT_NAME,
+    azureOpenAIApiVersion: process.env.AZURE_OPENAI_API_VERSION,
```

Comment:
will it still work with my default settings if I don't set any of these?

Reply:
Yes, it works well when using the original OpenAI.
LangChain 0.0.70 or later uses either the original or Azure, whichever is set.